prac_08/silver_service_taxi.py

```diff
@@ -0,0 +1,25 @@
+"""
+CP1404/CP5632 Practical
+Taxi class
+"""
+from prac_08.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    """Represent a SilverServiceTaxi."""
+    FLAGFALL = 4.50
```

Comment:
This should not be a constant. We might want to change it.

Reply:
I have now changed this. Thank you.